### PR TITLE
front: drop store related to tsv1

### LINKED ISSUE #8065: Drop Store TSV1
### Description and goal

Remove all code in store related to the old model

Actions to remove (not exhaustive):
- [x] updateViaStopTime
- [x] updateOrigin
- [x] updateDestination
- [x] addVias
- [x] permuteVias
- [x] deleteVias
- [x] replaceVias
- [x] clearVias
- [x] updatePathfindingID
- [x] updateSuggeredVias
- [x] updatePowerRestrictionRanges
- [x] updateAllowances
- [x] updateItinerary
- [x] deleteItinerary
- [x] updateDepartureTime
- [x] updateRollingStockComfortType
- [x] updateTrainScheduleIdToModify
- [x] updateDestinationTime (from old stdcm, not used anymore)
- [x] updateDestinationDate (from old stdcm, not used anymore)
- [x] updateSimulation (and all undo/redo etc)
- [x] updateSelectedProjection
- [x] updateAllowancesSettings
- [x] updateSpeedSpaceSettings
- [x] updateSignalBase

Selectors to remove (not exhaustive):
- [x] getVias
- [x] getOrigin
- [x] getDestination
- [x] getGeojson
- [x] getSuggeredVias
- [x] getPathfindingId
- [x] getPowerRestrictionRanges
- [x] getAllowances
- [x] getDepartureTime
- [x] getRollingStockComfortType
- [x] getTrainScheduleIdToModify
- [x] getDestinationTime (from old stdcm, not used anymore)
- [x] getDestinationDate (from old stdcm, not used anymore)
- [x] getSelectedProjection
- [x] getPresentSimulation
- [x] getAllowancesSettings
- [x] getSpeedSpaceSettings
- [x] getConsolidatedSimulation

Other:
- simulation.ts

### Acceptance criteria

- [ ] Old types are removed
- [ ] Old selectors are removed
- [ ] Old actions are removed
- [ ] Unused types in reducers/osrdsimulation are removed
- [ ] Check all DROP TSV1 or TODO TS2

### PR DESCRIPTION
**Pull request by SharglutDev**
close #8065 

To prevent some errors in warped map (bugged since drop switch, see issue [here](https://github.com/OpenRailAssociation/osrd/issues/8390)), some default values has been added in the components and should be replaced when fixing the issue.

To test : 
- We can still add multiple trains
- When adding/editing a train, the start time is still a date and not only a time
- Changing the projected train or the selected train keeps updating the simulation results

### CODE REVIEW

File: front/src/reducers/osrdsimulation/types.ts [hunk 1]
```diff
@@ -200,42 +193,11 @@ export interface SimulationTrain<DateType = Date> {
...
-  | SPEED_SPACE_SETTINGS_KEYS.CURVES
-  | SPEED_SPACE_SETTINGS_KEYS.MAX_SPEED
-  | SPEED_SPACE_SETTINGS_KEYS.SLOPES
-  | SPEED_SPACE_SETTINGS_KEYS.ELECTRICAL_PROFILES
-  | SPEED_SPACE_SETTINGS_KEYS.POWER_RESTRICTION;
-
-export type SpeedSpaceSettingsType = { [key in SpeedSpaceSettingKey]: boolean };
-
 export interface OsrdSimulationState {
   redirectToGraph?: boolean;
```
Comment (clarani) [inline front/src/reducers/osrdsimulation/types.ts]: You can remove redirectToGraph :)

**Review by clarani — APPROVED** [review]
Can you create new issues to:
-  replace RollingStockComfortType by the new enum Comfort
- to use slice in osrdSimulation

Otherwise, LGTM & tested :white_check_mark: 
Only left 1 comment, I think you can resolve it on your own :) Thanks !

**Comment by SharglutDev** [comment]
> Can you create new issues to:
> 
>     * replace RollingStockComfortType by the new enum Comfort
> 
>     * to use slice in osrdSimulation
> 
> 
> Otherwise, LGTM & tested ✅ Only left 1 comment, I think you can resolve it on your own :) Thanks !

- simulatio store refacto : https://github.com/OpenRailAssociation/osrd/issues/8412
- comfort type : there is already an [issue](https://github.com/OpenRailAssociation/osrd/issues/8254) I created last week to use this type in the rolling stocks endpoint so we are able to use it everywhere in front. Is is ok for you ?

File: front/src/reducers/osrdsimulation/types.ts (hunk 1 above)
Comment (SharglutDev) [reply front/src/reducers/osrdsimulation/types.ts]: Done :)

**Comment by clarani** [comment]
It's perfect, thanks :+1:

**Pull request merged**